Assets/Scripts/JumpPlayer.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class JumpPlayer : MonoBehaviour
+{
+    [SerializeField]
+    private string jumpButtonName;
+
+    [SerializeField]
+    private LayerMask ground;//This is how we know what the player can jump on
+
+    [SerializeField]
+    private float jumpHeight;
+
+    [SerializeField]
+    private float normalGravityMultiplier;//gravity on the way up
+
+    [SerializeField]
+    private float fallMultiplier;//gravity on the way down
+
+
+    private Rigidbody rigidbody;
+
+    private void Start()
+    {
+        rigidbody = GetComponent<Rigidbody>();
+    }
+    void Update ()
+    {
+        Debug.Log("On ground is: " + isOnGround().ToString());
+        Jump();
+    }
+
+    void Jump()
```

Comment:
The name is pretty clear here, but a ///<summary> block would add mouseover text to reference whenever it gets used elsewhere.